requirements.txt

```diff
@@ -0,0 +1,88 @@
+amqp==2.4.2
```

Comment:
Let's move file under `docker/girder`, until we figure out how to use this more broadly.